script/config.js

```diff
@@ -28,8 +29,19 @@ function getConfig() {
   const index = pathName.lastIndexOf('/')
   config.fileName = pathName.substr(index + 1)
 
-  const dir = os.tmpdir()
-  config.tempFile = path.join(dir, config.fileName)
+  const cacheDirEnv = process.env.DUGITE_CACHE_DIR
+
+  const cacheDir = cacheDirEnv
+    ? path.resolve(cacheDirEnv)
+    : os.tmpdir()
+
+  try {
+    fs.statSync(cacheDir)
+  } catch (e) {
```

Comment:
You might want to omit `e`, do not you?

Reply:
The omitting of the `e` is a [new TS feature](https://github.com/Microsoft/TypeScript/wiki/What's-new-in-TypeScript#optional-catch-clause-variables) that comes from a new ECMAScript feature, but because this is JS we can't do that just yet.

Reply:
I was not careful enough, and did not notice that this is a JS file and not a TS. Sorry for the noise.